wal-telegram

```diff
@@ -167,9 +167,14 @@ prepare() {
       # wal mode without tiled
       bg_path="$(<"${XDG_CACHE_HOME:-${HOME}/.cache}/wal/wal")"
       bg_file="${bg_path##*/}"
+      bg_file="${bg_file#.}" # if there is a '.' before the name
```

Comment:
I don't see why this is necessary? If there is a "." somewhere in the path before the filename it's going to get deleted with the previous line.

Reply:
In the case if the file is hidden (.bg). Without this line it will take the name of the file as extension (.bg -> some 'bg' extension) and will not execute the following lines with the `file` command.

Reply:
Ah right didn't think about that, seems good then. Thanks for the PR, I'll do the little change on the bg variable and merge.